mysql/mysqldump.go

```diff
@@ -186,24 +186,29 @@ func processCreateTable(conv *internal.Conv, stmt *ast.CreateTableStmt) {
 	var colNames []string
 	colDef := make(map[string]schema.Column)
 	var keys []schema.Key
+	var fkeys []schema.Fkey
```

Comment:
`fKeys`?

Reply:
foreign keys

Reply:
I was wondering should we change `fkeys` to `fKeys` according go's naming convention? Same for `spFKeys`.

Reply:
Done